docs/customization/tweaking-figures.ipynb

```diff
@@ -106,12 +155,68 @@
    "cell_type": "code",
    "execution_count": null,
    "id": "e7f8016d-1b89-4786-914c-6d45e76c91a1",
-   "metadata": {},
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
    "outputs": [],
    "source": [
     "pp.plot(da, title='This is my figure title')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "93c474b1-fcaa-4c96-b026-1e17ed278e72",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "# Further modifications\n",
```

Comment:
I think having a top-level section here (with a single '#') may mess up the table of contents in Sphinx.

How about having the following sections:
```
# Tweaking figures
  ## Basic customization
    ### Changing axes labels
    ### Setting the axis range
    ### Adding a title
  ## Further modifications
    ### Tick locations and labels
    ### Axes placement
```
?